backend/src/main/resources/liquibase/create-tables-changelog-1.0.xml

```diff
@@ -1238,4 +1238,86 @@
                                  referencedColumnNames="id"
                                  referencedTableName="keyvalue"/>
     </changeSet>
+
+    <changeSet author="lezhevg" id="new_lineage_id">
+        <createTable tableName="lineage_element">
+            <column name="id" type="int8" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_connection_entity">
+        <createTable tableName="lineage_connection">
+            <column name="element_id" type="int8">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="entity_id" type="int8">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="connection_type" type="int4">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="entity_type" type="int4">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_versioning_blob_entity">
+        <createTable tableName="lineage_versioning_blob">
```

Comment:
I think we need to add conditional on each one of these. @ravishetye ?

Reply:
example : https://github.com/VertaAI/modeldb/blob/master/backend/src/main/resources/liquibase/create-versioning-tables-changelog2.xml#L9-L13

Reply:
fixed